providers/gce/gce_fake.go

```diff
@@ -86,13 +88,14 @@ func NewFakeGCECloud(vals TestClusterValues) *Cloud {
 		regional:            vals.Regional,
 		networkURL:          vals.NetworkURL,
 		unsafeSubnetworkURL: vals.SubnetworkURL,
+		stackType:           vals.StackType,
 	}
 	c := cloud.NewMockGCE(&gceProjectRouter{gce})
 	gce.c = c
 	return gce
 }
 
-// UpdateFakeGCECloud updates the fake GCE cloud with the specified values. Currently only the onXPN value is updated.
-func UpdateFakeGCECloud(g *Cloud, vals TestClusterValues) {
```

Comment:
What's the reason for removing this function. Was it unused? This is an exported function so it may be used outside of this package and project.

Reply:
Most likely unused, but indeed maybe something depends on it, I restored it.